boa/src/builtins/array/mod.rs

```diff
@@ -33,6 +33,12 @@ use std::{
 pub(crate) struct Array;
 
 impl Array {
+    /// The name of the object.
+    pub(crate) const NAME: &'static str = "Array";
+
+    /// The amount of arguments this function object takes.
+    pub(crate) const LENGTH: i32 = 1;
```

Comment:
How come an i32 is chosen for the length?
Is there a reason it's signed?

Reply:
It's because `make_constructor_fn` and `make_builtin_fn` take a i32 as length, I think I'll change them to take in a `usize` so we don't have this problem